sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientOptions.java

```diff
@@ -21,6 +22,7 @@
     private Function<Duration, Duration> retryTimeout;
     private ProxyOptions proxyOptions;
     private HttpPipeline httpPipeline;
+    private HttpClient httpClient;
```

Comment:
I don't see this being used in this class. Since you are allowing users to set the HttpPipline, `httpClient` can be removed.

Reply:
With option 1 design, this is needed now, as we are allowing users to set the client too.